domain/lib/src/usecases/shared_space/shared_space_view_state.dart

```diff
@@ -386,6 +387,24 @@ class AddSharedSpaceMemberFailure extends FeatureFailure {
   List<Object> get props => [exception];
 }
 
+class UpdateSharedSpaceMemberViewState extends ViewState {
+  final SharedSpaceMember member;
```

Comment:
how about `updatedMember`?

Reply:
you're right, this `updatedMember` is redundant. Will remove it

Reply:
hmm, it's redundant. I will remove it